wasm/parsers/size.py

```diff
@@ -0,0 +1,10 @@
+from typing import IO
+
+from .integers import (
+    parse_u32,
+)
+
+
+def parse_size(stream: IO[bytes]) -> int:
```

Comment:
Is this to satisfy the type checker?  If so, would it be more appropriate to use `cast` instead of creating a whole new `int` object from an existing one?

Reply:
I believe I may have actually hit runtime errors when I fed a `numpy` integer type to `stream.read` but I'll double check.  If there are no runtime errors, I'll convert to `cast`

Reply:
No runtime errors, converted to a `cast`.